author.tf

```diff
@@ -498,3 +499,41 @@ module "author-dynamodb" {
 output "author_service_address" {
   value = "${module.author.service_address}"
 }
+
+resource "aws_lb_listener_rule" "listener_rule" {
```

Comment:
Why is this needed?

Reply:
It creates the LB listener rule to route traffic for `author.ons.gov.uk` to Author's target group.

Reply:
I'm probably being dumb here but isn't this covered by the other rules not requiring a specific host header?

Reply:
The host headers were only removed for the API.